test/ex_doc/retriever_test.exs

```diff
@@ -250,6 +250,19 @@ defmodule ExDoc.RetrieverTest do
       assert upcase.doc == ExDoc.Markdown.to_ast("See `String.upcase/1`.")
     end
 
+    test "signatures", c do
```

Comment:
I'm not sure if we should have this as a separate test or merge this with an existing one, e.g.:

https://github.com/elixir-lang/ex_doc/blob/47ddfd27f1172f43e5de49d4ea9708eae997da8e/test/ex_doc/retriever_test.exs#L15

wdyt @josevalim?

Reply:
I am working on a different PR where I am testing aiming to test all the Typespecs described in the Typespecs page. As I have found that some were not covered. So these will be added here as well.

Reply:
actually I'm keen to merge this as is, is that ok? I plan to move tests around after #1365 1362 lands.

Reply:
Sure. Please go ahead.